src/ansi-c/expr2c.cpp

```diff
@@ -104,7 +104,8 @@ void expr2ct::get_shorthands(const exprt &expr)
       it++)
   {
     const symbolt *symbol;
-    bool is_param=!ns.lookup(*it, symbol) && symbol->is_parameter;
+    bool is_param =
+      ns && !ns->get().lookup(*it, symbol) && symbol->is_parameter;
```

Comment:
This will mean the next step always is a `continue` - maybe all of `get_shorthands` should just immediately return?